modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptRxjsClientCodegen.java

```diff
@@ -89,8 +89,6 @@ public void setNpmRepository(String npmRepository) {
     @Override
     public void processOpts() {
         super.processOpts();
-        additionalProperties.put("isOriginalModelPropertyNaming", getModelPropertyNaming().equals("original"));
-        additionalProperties.put("modelPropertyNaming", getModelPropertyNaming());
```

Comment:
This was not used either in the class or in the mustache templates. Most likely - copy/paste from another generator with runtime conversion support (like typescript-fetch)